examples/solvers/douglas_rachford_pd_tomography_tv.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Total variation sparse angle tomography using the Douglas-Rachford solver.
+
+Solves the optimization problem
+
+    min_{0 <= x <= 1, Ax = g} lam ||grad(x)||_x
+
+where ``A`` is ray transform operator, ``g`` the given noisy data, ``eps`` is
+a small number, ``grad`` is the spatial gradient and || . ||_x is the so called
+cross-norm, giving rise to isotropic Total Variation.
+
+We do this by rewriting the problem on the form
```

Comment:
Do what? The beginning speaks of "solves", which I read as "this example solves", not as "we solves".
You may want to write "This problem can be rewritten in the form" here.